pos_return_order/README.rst

```diff
@@ -0,0 +1,101 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+PoS - Return Order
+==================
+
+This module was written to extend the functionality of odoo Point Of Sale about
```

Comment:
This module extends the functionality of odoo Point Of Sale about POS Order returns.